File: copy_logo.js
const fs = require('fs');
try {
  fs.copyFileSync(
    "C:\\Users\\LENOVO\\.gemini\\antigravity-ide\\brain\\bfec33b3-1019-4f00-bbc5-a1be059f9f52\\media__1779885401188.jpg",
    "d:\\work update app\\mobile_app\\assets\\logo.jpg"
  );
  console.log("Success");
} catch (e) {
  console.error("Error", e);
}

File: copy_logo.dart
import 'dart:io';

void main() {
  try {
    final sourcePath = r'C:\Users\LENOVO\.gemini\antigravity-ide\brain\a1f70f4f-b3fa-4396-93cf-9341234f786e\media__1788678205664.png';
    final targetPath = r'd:\work update app\mobile_app\assets\folk_logo.png';
    File(sourcePath).copySync(targetPath);
    stdout.writeln('Successfully copied folk logo to assets/folk_logo.png');
  } catch (e) {
    stderr.writeln('Error copying logo: $e');
  }
}
